src/components/CLI/CLI.tsx

```diff
@@ -23,12 +28,26 @@ export default function Cli({ decreaseCommandsLeft }: CliProps) {
     }
   };
 
+
   useEffect(() => {
     if (terminalRef.current) {
       terminalRef.current.scrollTop = terminalRef.current.scrollHeight;
     }
   }, [output]);
 
+  //Load initial command history if present
+  useEffect(() => {
+    const savedHistory = sessionStorage.getItem('terminalHistory');
```

Comment:
let's surround this with a try/catch?

Reply:
I guess we don't need try catch here, there is no possibility of sessionStorage not resolving, it's either data is there or not and both the cases are being handled